src/proteus/proteus.py

```diff
@@ -242,7 +239,7 @@ def start(self, *, resume: bool = False):
                 solvevol_target[e] = hf_row[e + "_kg_total"]
 
         # Download basic data
-        download_basic()
+        download_sufficient_data(self.config)
```

Comment:
Love this function name 😅

Reply:
Haha - thanks. It went through several iterations before reaching this point